pkg/k8s/node.go

```diff
@@ -135,7 +136,13 @@ func ParseNode(k8sNode *slim_corev1.Node, source source.Source) *nodeTypes.Node
 	}
 
 	newNode.Labels = k8sNode.GetLabels()
-	newNode.Annotations = k8sNode.GetAnnotations()
+	newNode.Annotations = make(map[string]string)
+	// Propagate only the annotations used for BGP Control Plane
```

Comment:
I wonder if we could loosen this and propagate everything with our prefix (`cilium.io/`) instead?

Reply:
Makes sense to me. I'll extend this to `*.cilium.io/`, so that it includes also possibly scoped labels.